tutorials/csvformatter/README.md

```diff
@@ -0,0 +1,64 @@
+##CsvFormatter Example
+
+`CsvFormatter` converts the incoming POJO to CSV(by default) or custom('|', ':' etc.) separated string.  
```

Comment:
CsvFormatter` converts the incoming POJO to CSV(by default) or custom delimiter ('|', ':' etc.) separated string.